docs/pages/rmf-audio-capture_TestSpec.md

```diff
@@ -0,0 +1,107 @@
+# RMF Audio Capture Test Specification Documentation
+
+## Table of Contents
+
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [Introduction](#introduction)
+- [Test Scenarios](#test-scenarios)
+  - [Check primary audio capture](#check-primary-audio-capture)
+  - [Check auxiliary audio capture](#check-auxiliary-audio-capture)
+  - [Check concurrent audio capture](#check-concurrent-audio-capture)
+
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` - Hardware Abstraction Layer, may include some common components
+- `UT` - Unit Test(s)
+- `SoC` - System on a Chip
+- `HAL` - Hardware Abstraction Layer
+- `API` - Application Programming Interface
+- `L2` - Level2 Testing
+- `L3` - Level3 Testing
+- `NA` - Not Applicable
+- `Y` - Yes
+
+
+## Introduction
+
+This document provides an overview of the testing requirements for the RMF audio capture module. It outlines the scope of testing, objectives, testing levels and approaches, specific test requirements, and expected deliverables.
+
+- RMF Audio Capture `HAL` specification Document \- <https://github.com/rdkcentral/rdk-halif-rmf_audio_capture/blob/main/docs/pages/rmf-audio-capture_halSpec.md>
```

Comment:
if you can update 
HAL specification is available here: [rmf-audio-capture_halSpec.md](https://github.com/rdkcentral/rdk-halif-rmf_audio_capture/blob/main/docs/pages/rmf-audio-capture_halSpec.md)
HAL interface is available here: [mfAudioCapture Header](https://github.com/rdkcentral/rdk-halif-rmf_audio_capture/blob/main/include/rmfAudioCapture.h)

Reply:
Updated